### ISSUE TITLE
arrows come and go and come and go

### ISSUE THREAD

**Issue by dmcdivitt** [Extended-Thunder/folder-pane-view-switcher#11]
I donated to the GoFundMe, Kickstarter, or whatever it was.

One day I see arrows to change folder views. The next day the arrows are gone. They may appear again. Then they leave again. Updates may be going out to cause this. I don't know. All I know is, I would like the extension to work. Right now no arrows show and have not for several days. TB 78.4.3 (32-bit).

**Comment by dmcdivitt** [comment]
To experiment I downloaded the extension and added as file. The arrows appeared! Joy! Then I closed TB and reopened. The arrows no longer appeared. Sadness. So the arrows appear when the extension is first installed. For instance when an update is available and TB automatically updates, the arrows appear because the extension was just installed. But upon closing TB and reopening the arrows are gone.

It would seem something does not persist after installation, closing, then reopening.
